docs/elements.md

```diff
@@ -244,6 +244,10 @@ To compute `max-select`, we use a similar algorithm (note the different default
 
 ### `pl-dropdown` element
 
+!!! warning
```

Comment:
Should we also shift the `pl-dropdown` element into the "Deprecated Elements" section?

Reply:
I would be fine with that move, if you're ok with that.

Reply:
Yep, it seems like that's what we want? @nwalters512 @eliotwrobson ?

Reply:
Done. Should I replace the uses of pl-dropdown in the example course as well?

Reply:
I'd say yes, with the caveat that I don't think they're compatible with each other so this would break any existing instances of the questions that use `<pl-dropdown>`.

Reply:
Maybe hold off on that transition for a bit? See #10286.

Reply:
Example course questions using pl-dropdown have been converted, except for two questions that use overlay (https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/questions/demo/overlayDropdown/question.html and https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/questions/demo/annotated/engLogic/question.html).